src/Routes.tsx

```diff
@@ -23,17 +23,47 @@ import { LoggetUtPage } from './component/page/LoggetUtPage'
 interface AppRoutesProps {
 	isLoading: boolean
 	isRejected: boolean
-	harTilgangTilArrangor: boolean
+	roller: string[]
 }
 
-export const AppRoutes = ({ isLoading, isRejected, harTilgangTilArrangor }: AppRoutesProps) => {
+export const AppRoutes = ({ isLoading, isRejected, roller }: AppRoutesProps) => {
 	if (isLoading) return <SpinnerPage/>
 	if (isRejected) return <ErrorPage/>
-	else if (!harTilgangTilArrangor) return <IngenRolleRoutes/>
-	return <PrivateRoutes/>
+	else if (roller.includes('KOORDINATOR') && roller.includes('VEILEDER')) return <VeilederOgKoordinatorRoutes/>
+	else if (roller.includes('KOORDINATOR')) return <KoordinatorRoutes/>
+	else if (roller.includes('VEILEDER')) return <VeilederRoutes/>
+	return <IngenRolleRoutes/>
 }
 
-const PrivateRoutes = (): React.ReactElement => {
+const KoordinatorRoutes = (): React.ReactElement => {
+	return (
+		<>
+			<Driftsmelding />
+			<Routes>
+				<Route path={DELTAKER_DETALJER_PAGE_ROUTE} element={<DeltakerDetaljerPage />} />
+				<Route path={GJENNOMFORING_DETALJER_PAGE_ROUTE} element={<GjennomforingDetaljerPage />} />
+				<Route path={GJENNOMFORING_LISTE_PAGE_ROUTE} element={<GjennomforingListePage />} />
+				<Route path={LEGG_TIL_DELTAKERLISTE_PAGE_ROUTE} element={<LeggTilDeltakerlistePage />} />
+				<Route path={DU_ER_LOGGET_UT_PAGE_ROUTE} element={<LoggetUtPage/>}/>
+				<Route path="*" element={<Navigate replace to={GJENNOMFORING_LISTE_PAGE_ROUTE}/>} />
+			</Routes>
+		</>
+	)
+}
+
+const VeilederRoutes = (): React.ReactElement => {
+	return (
+		<>
+			<Driftsmelding />
+			<Routes>
+				<Route path={INGEN_ROLLE_PAGE_ROUTE} element={<IngenRollePage />} />
+				<Route path="*" element={<Navigate replace to={INGEN_ROLLE_PAGE_ROUTE}/>} />
+			</Routes>
+		</>
+	)
+}
+
+const VeilederOgKoordinatorRoutes = (): React.ReactElement => {
```

Comment:
Det er jo mulig å trekke ut routene til koordinator til en liste eller eget komponent, osv. med veileder og felles routes også sy de sammen her. Fint å slippe å gjøre 2 endringer i steden for en, men synes ikke det har mye å si her.

Reply:
Ja, jeg er usikker på om det er verdt jobben egentlig :/ Det er jo enkelt å endre og enkelt å lese når det er to separate sånn som nå..

Reply:
Bare å unresolve her hvis du lander på at du vil at jeg skal endre på det :)